selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/ecms/ECMSLocator.java

```diff
@@ -1215,4 +1215,8 @@
   public static final SelenideElement ELEMENT_CHECK = $(byClassName("uiCheckbox"));
 
   public static final SelenideElement ELEMENT_BUTTON_CLOSE_SEARCH = $(byXpath("//*[@id=\"UIPopupWindow\"]/div[1]/a"));
+
+
+  public static final By ELEMENT_DELETE_POPUP_BUTTON =
+          By.xpath("//*[contains(@class, 'uiAction')]//*[text()='Delete']");
```

Comment:
Elements should not be selected by xpath, but rather by id or classes.

Reply:
ok! I will correct it